packages/youtube_dl.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Youtube_dl < Package
```

Comment:
I think you don't fully understand the ramifications of renaming packages.  Now users with `youtubedl` installed will not be able to get updates.  Probably makes sense to make youtubedl a fake package instead of removing it because you don't like the name.

Reply:
Okay, I'll do that. We need a better way to rename packages, would you mind working on that?

Reply:
We need a cleanup function in crew. If a package exists in the device,json file but no package file exists, perhaps we could see if @pkg.provides&.includes? ('oldpackagename') and if so rename oldpackage.{filelist,dlist} to newpackage.{filelist,dlist} if that doesn't clobber the new filelist,dlist, and if does clobber, delete the files in the old filelist, remove the old package from device.json,a nd reinstall the new package...

Reply:
An easier solution would be to just not rename the package. :)

Reply:
The easiest way in the short term is not always the easiest way in the long term.